ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.7__4.4.yaml

```diff
@@ -13,9 +13,6 @@ images:
 - context_dir: knative-operator
   from: base
   to: knative-operator
-- context_dir: olm-catalog/serverless-operator
```

Comment:
Why are you removing this part? is this related to the move from template to multi-stage?

Reply:
serverless-bundle fails to build since the release-1.7 branch does not contain the new bundle format. I'm not sure why this used to work.. didn't fully investigate yet.